.unreleased/feature_5875

```diff
@@ -0,0 +1 @@
+Implements: #5875 Add job information to log
```

Comment:
job information sounds rather broad...the patch adds exit status + runtime

Reply:
Changed it to "Add job exit status and runtime to log"